open-metadata-implementation/access-services/subject-area/subject-area-api/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/properties/objects/graph/Line.java

```diff
@@ -92,35 +85,44 @@ public Line(String name, LineType lineType) {
         this.setLineEnds();
     }
     protected void setLineEnds() {
-        OpenMetadataTypesArchiveAccessor archiveAccessor = OpenMetadataTypesArchiveAccessor.getInstance();
-
-        RelationshipDef relationshipDef =archiveAccessor.getRelationshipDefByGuid(this.typeDefGuid);
-        RelationshipEndDef endDef1 = relationshipDef.getEndDef1();
-        RelationshipEndDef endDef2 = relationshipDef.getEndDef2();
-
-        String end1TypeName = endDef1.getEntityType().getName();
-        String end2TypeName = endDef2.getEntityType().getName();
+//        OpenMetadataTypesArchiveAccessor archiveAccessor = OpenMetadataTypesArchiveAccessor.getInstance();
+//
+//        RelationshipDef relationshipDef =archiveAccessor.getRelationshipDefByGuid(this.typeDefGuid);
+//        RelationshipEndDef endDef1 = relationshipDef.getEndDef1();
+//        RelationshipEndDef endDef2 = relationshipDef.getEndDef2();
+//
+//        String end1TypeName = endDef1.getEntityType().getName();
+//        String end2TypeName = endDef2.getEntityType().getName();
         // Maybe consider subtypes here? We cannot use the Repositoryhelper isTypeOf because Line is used client side
         // which does not have access to the repository helper.
-        if (end1TypeName.equals("GlossaryTerm")) {
-            end1TypeName = "Term";
-        }
-        if (end2TypeName.equals("GlossaryTerm")) {
-            end2TypeName = "Term";
-        }
+//        if (end1TypeName.equals("GlossaryTerm")) {
+//            end1TypeName = "Term";
+//        }
+//        if (end2TypeName.equals("GlossaryTerm")) {
+//            end2TypeName = "Term";
+//        }
+//        getLineEnd1();
+//        getLineEnd1();
+//
+//        LineEnd lineEnd1 = new LineEnd(getEnd1TypeName(),
+//                                       getEnd1AttributeName(),
+//                                       getEnd1AttributeDescription(),
+//                                       endDef1.getAttributeCardinality()
+//                                       );
+//        LineEnd lineEnd2 = new LineEnd(end2TypeName,
+//                                       endDef2.getAttributeName(),
+//                                       endDef2.getAttributeDescription(),
+//                                       endDef2.getAttributeCardinality()
+//                                       );
```

Comment:
Should we leave this commented-out code?

Reply:
@immmus good spot :-).

Reply:
> @immmus good spot :-).

fixed